src/docs/cookbook/persistence/sqlite.md

```diff
@@ -268,21 +283,22 @@ Future<void> updateDog(Dog dog) async {
     'dogs',
     dog.toMap(),
     // Ensure that the Dog has a matching id.
-    where: "id = ?",
+    where: 'id = ?',
     // Pass the Dog's id as a whereArg to prevent SQL injection.
     whereArgs: [dog.id],
   );
 }
 ```
 
-<!-- skip -->
+<?code-excerpt "lib/main.dart (update2)"?>
 ```dart
-// Update Fido's age.
-await updateDog(Dog(
-  id: 0,
-  name: 'Fido',
-  age: 42,
-));
+// Update Fido's age and save it to the database.
+fido = Dog(
+  id: fido.id,
+  name: fido.name,
+  age: fido.age + 7,
+);
```

Comment:
In the full example, it reads the original value and increases it, I included that in the extracted code excerpt but I am happy to change it